docs/nodes/full-node/update-fullnode-with-new-releases.md

```diff
@@ -1,47 +1,72 @@
 ---
-title: "Update Fullnode With New Devnet Releases"
+title: "Update Fullnode With New Releases"
 slug: "update-fullnode-with-new-devnet-releases"
 sidebar_position: 11
 ---
 
 # Update Fullnode With New Releases
 
-When `devnet` is wiped and updated with newer versions, you will need to update your fullnode as well. If you do not, your fullnode will not continue to synchronize with the network. To update your fullnode, follow these steps:
+This document outlines the process for updating your fullnode. For fullnodes running in `devnet`, an additional data wipe step is required as `devnet` is wiped on every release.
 
 ## If you built the fullnode from aptos-core source code
 
-1. Shutdown your fullnode.
+1. Stop your fullnode by running the below command:
 
-2. Delete the data folder (the directory path is what you specified in the configuration file, e.g., `fullnode.yaml`).
+```bash
+cargo stop aptos-node
+```
```

Comment:
This works?  That's kinda cool.  Hopefully it doesn't have too many dependencies / take a long time to compile to run first.